roles/common/templates/ubuntu.list

```diff
@@ -0,0 +1,9 @@
+# Ubuntu Main Archive
```

Comment:
Even if they are in the documentation, they are the default ubuntu repositories. Moreover overwritting `/etc/apt/sources.list` is not recommended as it can include other repositories.

For static files you should use `files` instead of `templates`